skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkiaLayer.jvm.kt

```diff
@@ -92,13 +94,31 @@ actual open class SkiaLayer internal constructor(
         }
     }
 
+    private var fullscreenDispatcher = object : ComponentAdapter() {
+        var isFullscreenDispatched: Boolean = false
+        override fun componentShown(e: ComponentEvent) {
```

Comment:
Is componentShown mandatory? Changing fullscreen doesn't work in addNotify?

When we in `addNotify`, we already have the native window created. If we change fullscreen only after it is showed, we will have fullscreen change glitch at the start of the application.